app/cmd/rpc/mesh/relay.go

```diff
@@ -228,9 +228,9 @@ func notifyServicer(r *pocketTypes.Relay) {
 	} else {
 		LogRelay(r, "notify - servicer processed relay successfully", LogLvlDebug)
 
-		exhausted := ns.CountRelay()
+		canHoldMore := ns.CountRelay()
```

Comment:
so this was a small bug?

Reply:
was bug because the function return an evaluation in sense a to b and the if evaluate it on the revser way